extensions/metriport/actions/patient/create.ts

```diff
@@ -12,6 +11,11 @@ import { createFields } from './fields'
 import { stringId } from '../../validation/generic.zod'
 import { type PatientCreate, patientCreateSchema } from './validation'
 import { patientIdDataPoint } from './dataPoints'
+// import { isValid } from 'driver-license-validator'
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+const isValid = (a: string, b: any) => {
+  return true
+}
```

Comment:
thanks, i should've provided additional comments around this.

i requested a change to metriport. there was an issue with one of its dependencies, and it should be solved in [this pr](https://github.com/awell-health/awell-extensions/pull/129), which will allow us to revert these lines.

i kept it like this in case ppl wanted to test on local.